### DISCUSSION TITLE
When will the latest beta docker images be published to Docker Hub?

### DISCUSSION THREAD

**Discussion by nicari92** [blakeblackshear/frigate#5224] [General]
I noticed that on GitHub the latest beta version is the 0.12.0-beta5, but on Docker Hub the latest available beta is the 0.12.0-beta2.
I tried to build my own image, but i I wasn't able to do it. I'd like to test the latest releases, but without the Docker image I can't.

**Comment by trythatagain** [comment]
I don't think they are going to use Docker hub going forward. I did see the announcement, but can't remember whereabouts in the discussion list.

**Comment by blakeblackshear** [comment]
We are no longer publishing images to dockerhub starting with beta3. GHCR is much easier to work with and provides stats for each tag. 

> We have decided to stop publishing builds to Dockerhub and new builds will be available exclusively on GHCR going forward. Please update to use the new image locations by adding `ghcr.io/` in front of your image names. (`blakeblackshear/frigate:0.12.0-beta3` needs to be changed to `ghcr.io/blakeblackshear/frigate:0.12.0-beta3`)

https://github.com/blakeblackshear/frigate/releases/tag/v0.12.0-beta3

**Reply by nicari92** [reply]
This worked! Thanks! I missed the migration to GHCR in the release notes
